src/server/api/schema.js

```diff
@@ -1155,11 +1165,15 @@ const rootResolvers = {
         return user;
       }
     },
+    currentUserWithAccess: async (_, { organizationId, role }, { user }) => {
+      await accessRequired(user, organizationId, role, false);
+      return user;
+    },
     organizations: async (_, { id }, { user }) => {
       if (user.is_superadmin) {
         return r.table("organization");
       } else {
-        return await cacheableData.user.userOrgs(user.id, "TEXTER");
+        return await cacheableData.user.userOrgs(user.id, "SUSPENDED");
```

Comment:
INFO (no critique): I think this will (or could help) fix a long-standing bug where sometimes people get a "you are not affiliated with any organizations"